database_scripts/tabels/create_customer.sql

```diff
@@ -1,12 +1,14 @@
 use customer_db
 
-create table Customer
-(
-ID int  NOT NULL PRIMARY KEY,
-UserID int UNIQUE FOREIGN KEY dbo.User
-[Name] nvarchar(100),
-Email nvarchar(250),
-[Street] nvarchar (250),
-PostCode int,
-Country nvarchar(250),
-)
\ No newline at end of file
+CREATE TABLE [dbo].[Customers] (
+    [ID] [int] NOT NULL IDENTITY,
+    [UserID] [int],
+    [FirstName] [nvarchar](max),
+    [LastName] [nvarchar](max),
+    [UserName] [nvarchar](max),
+    [Email] [nvarchar](max),
+    [Phone] [nvarchar](max),
+    [UpdatedDate] [datetime],
+    [CreatedDate] [datetime],
+    CONSTRAINT [PK_dbo.Customers] PRIMARY KEY ([ID])
+)
```

Comment:
Tu jest ok, tylko musisz dodać NOT NULL przy polach, które są IsRequired() i długości nvarcharów np. [FirstName] [nvarchar](100) oraz zmień `use customer_db` na `use BProject_db`